applications/crossbar/priv/couchdb/schemas/access_lists.json

```diff
@@ -0,0 +1,33 @@
+{
+  "_id": "access_lists",
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
The jesse JSON validator only handles draft-03 schemas. Please change this otherwise jesse won't validate the data using this schema doc.